### DISCUSSION TITLE
Saludos buenas noches, estoy comenzando a conocer la plataforma de GibHub.

### DISCUSSION THREAD

**Discussion by Cleto85** [community/community#147576] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Saludos, es la primera vez que pido ayuda para saber cómo cambio el idioma en español dentro de mi cuenta de usuario en github

**Comment by arteze** [comment]
Es algo que quise desde hace años, pero parece que no se puede, porque todavía no lo tradujeron al español.

**Comment by mecodeatlas** [comment]
Hi @Cleto85, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.

Oh, and you are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
